### PR TITLE
[Workers] alt text for images

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/workers/learning/using-services/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You can create and switch between environments in the dashboard:
 2.  Choose your **Worker**.
 3.  Select **production** > **Create environment**.
 
-![production-button](./media/create-environment.png)
+![Selecting the Create environment button to begin configuring a new environment](./media/create-environment.png)
```
Comment (deadlypants1973) [inline content/workers/learning/using-services/index.md]:
```suggestion
![Select Create environment to begin configuring a new environment](./media/create-environment.png)
```

File: content/workers/learning/using-services/index.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ Unlike Wrangler environments, Workers Service environments do not create extra W
 3.  Choose your **Worker**.
 4.  Select **Quick edit**.
 
-![Quick edit](./media/quick-edit.png)
+![Selecting the Quick edit button to make edits to the Worker script](./media/quick-edit.png)
```
Comment (deadlypants1973) [inline content/workers/learning/using-services/index.md]:
```suggestion
![Select Quick edit to make edits to the Worker script](./media/quick-edit.png)
```

File: content/workers/platform/cron-triggers/index.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ A recommended way for testing your Cron Trigger is to first deploy it to a test
 
 Users can review the execution history of their Cron Triggers in **Past Events** under [**Triggers**](https://dash.cloudflare.com/?to=/:account/workers) or through Cloudflare's [GraphQL Analytics API](/analytics/graphql-api).
 
-![workers-past-events](./media/workers-past-events.png)
+![In Past Events, you can review the activity log of past cron triggers that you set up](./media/workers-past-events.png)
```
Comment (deadlypants1973) [inline content/workers/platform/cron-triggers/index.md]:
```suggestion
![Review the activity log of past cron triggers in Past Events](./media/workers-past-events.png)
```

File: content/workers/platform/deploy-button.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Deploy buttons let you deploy projects to the Workers platform in under five min
 
 Try the deploy button below to deploy a GraphQL server:
 
-[![Deploy to Cloudflare Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/signalnerve/workers-graphql-server)
+[![Select this button to deploy a GraphQL server](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/signalnerve/workers-graphql-server)
```
Comment (dcpena) [inline content/workers/platform/deploy-button.md]: maybe swap "this" for "Deploy with Workers" so it's clearer for alt text?

File: content/workers/platform/deploy-button.md [hunk 5]
```diff
@@ -56,4 +56,4 @@ jobs:
 [![Deploy to Cloudflare Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/YOURUSERNAME/YOURREPO)
 ```
 
-[![Deploy to Cloudflare Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com)
+[![Select this button to be redirected to the Deploy with Workers page](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com)
```
Comment (dcpena) [inline content/workers/platform/deploy-button.md]: same comment as above for "Deploy with Workers" text

File: content/workers/tutorials/authorize-users-with-auth0/index.md [hunk 6]
```diff
@@ -987,7 +987,7 @@ Wrangler will compile your code, upload the associated Workers Sites folder (`pu
 3.  Let Auth0 redirect you to `/auth`, and then to `/`. As this is happening, your Workers application has exchanged a login `code` with Auth0 for an access token, persisted it to Workers KV, and registered you as an authorized user via a cookie.
 4.  If you see your site, you have successfully authorized users to your Workers application, using Auth0.
 
-![Example GIF](./media/example-vid.gif)
+![After successfuly setup, you will be able to log into your application with Auth0's login page](./media/example-vid.gif)
```
Comment (dcpena) [inline content/workers/tutorials/authorize-users-with-auth0/index.md]: successful?

File: content/workers/tutorials/deploy-a-static-wordpress-site/index.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ From the **Where will you host...** dropdown on the page, select _ZIP archive (.
 
 To do your first export, select **Start static site export** (this operation may take a while), and when it completes, save the ZIP file somewhere you can easily find later.
 
-![Exporting your WordPress site](./media/wordpress--export.png)
+![Exporting your WordPress site by  selecting Start static site export in the WordPress dashboard](./media/wordpress--export.png)
```
Comment (dcpena) [inline content/workers/tutorials/deploy-a-static-wordpress-site/index.md]:
```suggestion
![Exporting your WordPress site by selecting Start static site export in the WordPress dashboard](./media/wordpress--export.png)
```

File: content/workers/tutorials/store-data-with-fauna/index.md [hunk 8]
```diff
@@ -54,21 +54,21 @@ In the Fauna dashboard:
...
 
-![Creating your database in Fauna](./media/create-database.png)
+![Create your database in Fauna by setting a name and region group](./media/create-database.png)
 
 ### Creating the products catalog
 
 Select **New Collection** to create the **Products** collection that stores your inventory documents.
 
-![Creating your "Products" collection](./media/create-collection.png)
+![Create your Products collection by following direction above](./media/create-collection.png)
```
Comment (dcpena) [inline content/workers/tutorials/store-data-with-fauna/index.md]:
```suggestion
![Create your Products collection by following the directions above](./media/create-collection.png)
```

**Review by dcpena — APPROVED** [review]
Small bits of feedback but otherwise LGTM

File: content/workers/platform/deploy-button.md (hunk 4 above)
Comment (deadlypants1973) [inline content/workers/platform/deploy-button.md]:
```suggestion
[![Select Deploy with Workers to deploy a GraphQL server](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/signalnerve/workers-graphql-server)
```

File: content/workers/platform/deploy-button.md (hunk 5 above)
Comment (deadlypants1973) [inline content/workers/platform/deploy-button.md]:
```suggestion
[![Select Deploy with Workers to be redirected to the Deploy with Workers page](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com)
```

File: content/workers/tutorials/authorize-users-with-auth0/index.md (hunk 6 above)
Comment (deadlypants1973) [inline content/workers/tutorials/authorize-users-with-auth0/index.md]:
```suggestion
![After successful setup, you will be able to log into your application with Auth0's login page](./media/example-vid.gif)
```

**Pull request merged**